libbeat/outputs/elasticsearch/bulkapi.go

```diff
@@ -160,15 +161,10 @@ func bulkEncode(metaBuilder MetaBuilder, body []interface{}) bytes.Buffer {
 	return buf
 }
 
-func readBulkResult(obj []byte) (*BulkResult, error) {
+func readBulkResult(obj []byte) (BulkResult, error) {
 	if obj == nil {
```

Comment:
It looks like this `if` could be dropped since the effective return object will be the same in both cases.